controllers/userController.js

```diff
@@ -1,7 +1,7 @@
-const UserService = require("../services/UserService");
-const AuthService = require("../services/AuthService");
+const UserService = require("../services/userService");
+const AuthService = require("../services/authService");
 // const FollowService = require("../services/FollowService");
-const FollowFindPair = require("../services/FollowFindPair");
```

Comment:
the filename is correct here,

Reply:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/11273150/236553032-df81b7ad-8918-49c2-b8fb-96615d7039a0.png">